### PR TITLE
add SDE version into both footer and settings page

### PR DESCRIPTION
**Pull request by warlof**
Depends on https://github.com/eveseat/console/commit/5c7f85b8244d76e9cf957476119e9ba0747027c6

### CODE REVIEW

File: src/Http/Controllers/Configuration/SeatController.php [hunk 1]
```diff
@@ -70,4 +74,26 @@ public function postUpdateSettings(SeatSettings $request)
             ->with('success', 'SeAT settings updated!');
     }
 
+    /**
+     * Help to fetch latest approved SDE version file from resources repository
+     *
+     * @return string Json
+     */
+    private function getApprovedSDE()
```
Comment (leonjza) [inline src/Http/Controllers/Configuration/SeatController.php]: This method is going to block the request and increase the load time. It should become an ajax request like its handled with api keys.

File: src/lang/en/seat.php [hunk 2]
```diff
@@ -329,6 +329,7 @@
     'allow_sso'                    => 'Allow Single Signon',
     'allow_registration'           => 'Allow Registration',
     'module_versions'              => 'SeAT Module Versions',
+    'tp_versions'                  => 'Third Party Versions',
```
Comment (leonjza) [inline src/lang/en/seat.php]: Dont think this is relevant to this PR.

File: src/resources/views/configuration/settings/view.blade.php [hunk 3]
```diff
@@ -234,4 +234,24 @@ class="form-control input-md" value="{{ setting('analytics_id', true) }}" disabl
...
+      <h3 class="panel-title">{{ trans('web::seat.tp_versions') }}</h3>
+    </div>
+    <div class="panel-body">
+
+      <dl>
+
+        <dt>Eve Online SDE</dt>
+        <dd>
+          <ul>
+            <li>{{ trans('web::seat.installed') }}: <b>{{ Seat\Services\Settings\Seat::get('installed_sde') }}</b></li>
```
Comment (leonjza) [inline src/resources/views/configuration/settings/view.blade.php]: Use the [setting helper](https://github.com/eveseat/services/blob/master/src/Helpers/helpers.php#L187) please.

File: src/resources/views/includes/footer.blade.php [hunk 4]
```diff
@@ -3,6 +3,7 @@
   <!-- To the right -->
   <div class="pull-right hidden-xs">
     <b>{{ trans('web::seat.render_in') }}</b> {{ number((microtime(true) - LARAVEL_START), 3) }}ms |
+    <b>{{ trans('web::seat.sde_version') }}</b> {{ Seat\Services\Settings\Seat::get('installed_sde') }}
```
Comment (leonjza) [inline src/resources/views/includes/footer.blade.php]: Use the [setting helper](https://github.com/eveseat/services/blob/master/src/Helpers/helpers.php#L187) please.

File: src/Http/Controllers/Configuration/SeatController.php [hunk 5]
```diff
@@ -70,4 +74,26 @@ public function postUpdateSettings(SeatSettings $request)
...
+    {
+        $sde_uri = "https://raw.githubusercontent.com/eveseat/resources/master/sde.json";
+
+        $curl = curl_init();
```
Comment (leonjza) [inline src/Http/Controllers/Configuration/SeatController.php]: Please use Guzzle and not a raw curl request.

File: src/lang/en/seat.php (hunk 2 above)
Comment (warlof) [inline src/lang/en/seat.php]: I've add à version panel below Seat version panel where puting plugins and other third party related things.

Assuming SDE is third party

**Pull request merged**